storage-proto/src/convert.rs

```diff
@@ -717,6 +717,9 @@ impl From<TransactionError> for tx_by_addr::TransactionError {
                             InstructionError::AccountNotRentExempt => {
                                 tx_by_addr::InstructionErrorType::AccountNotRentExempt
                             }
+                            InstructionError::InvalidAccountOwner => {
```

Comment:
Also needs to be included in https://github.com/solana-labs/solana/blob/cf4e31964b1ed6e94f9abf4303d6816c3e498e39/storage-proto/src/convert.rs#L487-L497

BorshIoError and AccountNotRentExempt are also missing there. Do you mind including those as well?

Reply:
added, thanks